content/systems/filecoin_markets/retrieval_market/_index.md

```diff
@@ -13,89 +13,74 @@ dashboardTests: 0
 
 ## Components
 
-The `retrieval market` refers to the process of negotiating deals for a provider to serve stored data to a client.
+The `retrieval market` refers to the process of negotiating deals for a provider to serve stored data to a client. It should be highlighted that the negotiation process for the retrieval happens primarily off-chain. It is only some parts of it (mostly relating to redeeming vouchers from payment channels) that involve interaction with the blockchain.
 
 The main components are as follows:
 
-- A payment channel actor (see [Payment Channel Actor](payment_channel_actor) for details)
-- 2 `libp2p` services - 
-   - a protocol for making queries
-   - a protocol for negotiating and carrying out retrieval deals (will disappear as a seperate libp2p protocol as retrieval markets evolve)
+- A payment channel actor (see {{<link payment_channel_actor>}} for details)
+- A Data Transfer subsystem and protocol used to query retrieval miners and initiate retrieval deals
 - A chain-based content routing interface
-- A client module to query retrieval miners and initiate deals for retrieval
 - A provider module to respond to queries and deal proposals
 
-## VO & V1
+The retrieval market operate by piggybacking on the Data Transfer system and Graphsync to handle transfer and verification, to support arbitrary selectors, and to reduce round trips. The retrieval market can support sending arbitrary payload CIDs & selectors within a piece. 
 
-V0 of the protocol has participants send data over the retrieval protocol itself in a series of Blocks encoded in Bitswap format and verify received blocks manually. It will only support fetching the payload CID which is at the root of PieceCID's `.car` File, and will only support fetching the whole DAG.
+The Data Transfer System is augmented accordingly to support pausing/resuming and sending intermediate vouchers to facilitate this.
 
-In V1, the retrieval markets will evolve to support sending arbitrary payload CID's & selectors within a piece. Further, it will piggy back on the Data Transfer system and Graphsync to handle transfer and verification, to support arbitrary selectors, and to reduce round trips.
-The Data Transfer System will accordingly be augmented to support pausing/resuming and sending intermediate vouchers to facilitate this.
-V1 will also include additional mechanisms for timeouts and cancellations (to be specified).
+Additional features, which are yet to be specified include mechanisms for timeouts and cancellations.
```

Comment:
let's remove this sentence as we don't have immediate plans for timeouts & cancellations